documentation/asciidoc/topics/con_operator_deployments.adoc

```diff
@@ -34,9 +34,14 @@ endif::downstream[]
 
 = Resource reconciliation
 
-{ispn_operator} reconciles `Infinispan` custom resources such as the `Cache` CR with resources on your {brandname} cluster.
+{ispn_operator} reconciles custom resources such as the `Cache` CR with resources on your {brandname} cluster.
 
-Bidirectional reconciliation synchronizes your CRs with changes that you make to `Infinispan` resources through the {brandname} Console, command line interface (CLI), or other client application and vice versa.
+Bidirectional reconciliation synchronizes your CRs with changes that you make to {brandname} resources through the {brandname} Console, command line interface (CLI), or other client application and vice versa.
 For example if you create a cache through the {brandname} Console then {ispn_operator} adds a declarative Kubernetes representation.
 
-To perform reconciliation {ispn_operator} creates a `listener` pod in each {brandname} cluster that detects modifications for `Infinispan` resources.
+{ispn_operator} associates declarative Kubernetes representations of {brandname} resources with an `Infinispan` CR.
+Deleting an `Infinispan` CR also deletes the {brandname} resources.
+For example deleting an `Infinispan` CR removes any associated `Cache` CRs.
+However deleting an `Infinispan` CR does not remove any {brandname} resources that you create through the console, CLI, or client applications.
```

Comment:
This isn't quite right.

Deleting an `Infinispan` CR only removes Cache CRs that were created by the config listen, i.e. not Cache CRs created explicitly by the user.

Deleting an `Infinispan` CR most definitely removes the resources that you create through the console etc as it removes the underlying cluster they are created on :slightly_smiling_face:. It will also remove any Cache CR created by the config listener.

Reply:
:facepalm: I suppose it would help if I actually thought about it.